asv_bench/benchmarks/sparse.py

```diff
@@ -45,8 +45,7 @@ def time_sparse_array(self, dense_proportion, fill_value, dtype):
 class SparseDataFrameConstructor:
     def setup(self):
         N = 1000
-        self.arr = np.arange(N)
-        self.sparse = scipy.sparse.rand(N, N, 0.005)
+        self.sparse = scipy.sparse.rand(N, N, 0.005, random_state=0)
```

Comment:
given what we said in the other PR, you can remove this random state again (it's covered by the global setup function which will be run for every benchmark)